scripts/converter/convert_eun_to_nue.py

```diff
@@ -51,7 +51,14 @@ def convert_to_nue(filename):
                 node['fields'].append({'name': 'coordinateSystem',
                                        'value': 'NUE',
                                        'type': 'SFString'})
-        elif node['name'] not in ['Viewpoint', 'TexturedBackground', 'TexturedBackgroundLight']:
+        elif node['name'] == 'Viewpoint':
+            print('Rotating', node['name'])
+            for field in node['fields']:
+                if field['name'] in ['orientation']:
+                    field['value'] = rotation(field['value'], [0, 1, 0, 0.5 * math.pi])
+                elif field['name'] in ['position']:
+                    field['value'] = [field['value'][2], field['value'][1], str(-float(field['value'][0]))]
```

Comment:
Here we are missing a conversion when there is no orientation or position defined

Reply:
If no orientation or position are defined, I chose to set the `Viewpoint` to be as we would see it with the default parameters of Webots in a `NUE` world. So for EUN-->NUE nothing is done, and for NUE-->ENU the default Viewpoint is adapted.

Let me know if you have another idea of implementation!